src/js/components/award/contract/ContractDetails.jsx

```diff
@@ -4,14 +4,13 @@
  **/
 
 import React from 'react';
-import PropTypes from 'prop-types';
 import moment from 'moment';
 import DetailRow from '../DetailRow';
 
 const propTypes = {
-    selectedAward: PropTypes.object,
-    seeAdditional: PropTypes.func,
-    maxChars: PropTypes.number
+    selectedAward: React.PropTypes.object,
```

Comment:
Please use the new [PropTypes](https://github.com/facebook/prop-types#usage) module.